packages/webcomponents/src/components/business-form-stepped/business-core-info/business-core-info-form-step.tsx

```diff
@@ -48,101 +117,103 @@ export class BusinessCoreInfoFormStep {
-                name="doing_business_as"
-                label="Doing Business As (DBA)"
-                defaultValue={genericInfoDefaultValue.doing_business_as}
-                error={this.errors.doing_business_as}
-                inputHandler={this.inputHandler}
-              />
-            </div>
-            <div class="col-12 col-md-6">
-              <form-control-select
-                name="business_type"
-                label="Business Type"
-                options={BusinessTypeOptions}
-                defaultValue={genericInfoDefaultValue.business_type}
-                error={this.errors.business_type}
-                inputHandler={this.inputHandler}
-              />
-            </div>
-            <div class="col-12 col-md-6">
-              <form-control-select
-                name="business_structure"
-                label="Business Structure"
-                options={BusinessStructureOptions}
-                defaultValue={genericInfoDefaultValue.business_structure}
-                error={this.errors.business_structure}
-                inputHandler={this.inputHandler}
-              />
-            </div>
-            <div class="col-12">
-              <form-control-text
-                name="industry"
-                label="Industry"
-                defaultValue={genericInfoDefaultValue.industry}
-                error={this.errors.business_structure}
-                inputHandler={this.inputHandler}
-              />
-            </div>
-            <div class="col-12">
-              <form-control-number-masked
-                name="tax_id"
-                label="Tax ID"
-                defaultValue={genericInfoDefaultValue.tax_id}
-                error={this.errors.tax_id}
-                inputHandler={this.inputHandler}
-                mask={TAX_ID_MASKS.US}
-              />
-            </div>
-            <div class="col-12">
-              <form-control-text
-                name="website_url"
-                label="Website URL"
-                defaultValue={genericInfoDefaultValue.website_url}
-                error={this.errors.website_url}
-                inputHandler={this.inputHandler}
-              />
-            </div>
-            <div class="col-12 col-md-6">
-              <form-control-text
-                name="email"
-                label="Email Address"
-                defaultValue={genericInfoDefaultValue.email}
-                error={this.errors.email}
-                inputHandler={this.inputHandler}
-              />
-            </div>
-            <div class="col-12 col-md-6">
-              <form-control-number-masked
-                name="phone"
-                label="Phone Number"
-                defaultValue={genericInfoDefaultValue.phone}
-                error={this.errors.phone}
-                inputHandler={this.inputHandler}
-                mask={PHONE_MASKS.US}
-              />
+        <form onSubmit={() => this.sendData}>
```

Comment:
I think this might not be necessary since the `validateAndSubmit` method will be called

Reply:
Good call! Fixed